public/index.html

```diff
@@ -2,19 +2,19 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/multirotor-icon.png" />
```

Comment:
ico and png are different formats, if you want to use the icon, please use some online program to convert our png into an ico, name it "favicon.ico" and place it in the correct spot